lib/registry.dart

```diff
@@ -367,29 +367,41 @@ class RegistryManager {
   void _handleCommand(List command) {
     switch (command[0]) {
       case _addValue:
-        _add(command[1], command[2] as List, command[3] as SendPort);
+        _add(command[1], command[2] as List?, command[3] as SendPort);
```

Comment:
Consider using `as List<Object?>?` instead of just `as List?`. Similar for raw `List`s below.